test/libyul/YulOptimizerTest.cpp

```diff
@@ -128,312 +73,12 @@ TestCase::TestResult YulOptimizerTest::run(ostream& _stream, string const& _line
-	{
-		disambiguate();
-		// apply SSA
-		SSATransform::run(*m_context, *m_object->code);
-		RedundantAssignEliminator::run(*m_context, *m_object->code);
-		// reverse SSA
-		SSAReverser::run(*m_context, *m_object->code);
-		CommonSubexpressionEliminator::run(*m_context, *m_object->code);
-		UnusedPruner::run(*m_context, *m_object->code);
-	}
-	else if (m_optimizerStep == "stackCompressor")
-	{
-		disambiguate();
-		FunctionGrouper::run(*m_context, *m_object->code);
-		size_t maxIterations = 16;
-		Object obj;
-		obj.code = m_object->code;
-		StackCompressor::run(*m_dialect, obj, true, maxIterations);
-		m_object->code = obj.code;
-		BlockFlattener::run(*m_context, *m_object->code);
-	}
-	else if (m_optimizerStep == "wordSizeTransform")
-	{
-		disambiguate();
-		ExpressionSplitter::run(*m_context, *m_object->code);
-		WordSizeTransform::run(*m_dialect, *m_dialect, *m_object->code, *m_nameDispenser);
-	}
-	else if (m_optimizerStep == "fullSuite")
-	{
-		GasMeter meter(dynamic_cast<EVMDialect const&>(*m_dialect), false, 200);
-		yul::Object obj;
-		obj.code = m_object->code;
-		obj.analysisInfo = m_analysisInfo;
-		OptimiserSuite::run(*m_dialect, &meter, obj, true, solidity::frontend::OptimiserSettings::DefaultYulOptimiserSteps);
-	}
-	else if (m_optimizerStep == "stackLimitEvader")
-	{
-		yul::Object obj;
-		obj.code = m_object->code;
-		obj.analysisInfo = m_analysisInfo;
-		disambiguate();
-		StackLimitEvader::run(*m_context, obj, CompilabilityChecker{
-			*m_dialect,
-			obj,
-			true
-		}.unreachableVariables);
-	}
-	else if (m_optimizerStep == "fakeStackLimitEvader")
-	{
-		yul::Object obj;
-		obj.code = m_object->code;
-		obj.analysisInfo = m_analysisInfo;
-		disambiguate();
-		// Mark all variables with a name starting with "$" for escalation to memory.
-		struct FakeUnreachableGenerator: ASTWalker
-		{
-			map<YulString, set<YulString>> fakeUnreachables;
-			using ASTWalker::operator();
-			void operator()(FunctionDefinition const& _function) override
-			{
-				YulString originalFunctionName = m_currentFunction;
-				m_currentFunction = _function.name;
-				ASTWalker::operator()(_function);
-				m_currentFunction = originalFunctionName;
-			}
-			void visitVariableName(YulString _var)
-			{
-				if (!_var.empty() && _var.str().front() == '$')
-					fakeUnreachables[m_currentFunction].insert(_var);
-			}
-			void operator()(VariableDeclaration const& _varDecl) override
-			{
-				for (auto const& var: _varDecl.variables)
-					visitVariableName(var.name);
-				ASTWalker::operator()(_varDecl);
-			}
-			void operator()(Identifier const& _identifier) override
-			{
-				visitVariableName(_identifier.name);
-				ASTWalker::operator()(_identifier);
-			}
-			YulString m_currentFunction = YulString{};
-		};
-		FakeUnreachableGenerator fakeUnreachableGenerator;
-		fakeUnreachableGenerator(*obj.code);
-		StackLimitEvader::run(*m_context, obj, fakeUnreachableGenerator.fakeUnreachables);
-	}
-	else
+	if (!success)
```

Comment:
```suggestion
	if (!tester.runSetup)
```
and removing line 79?